src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -30,6 +31,9 @@
 @SuppressWarnings("ComponentNotRegistered")
 public class InstallSdkAction extends DumbAwareAction {
 
+  //TODO(pq): add support for "git.exe"
+  private static final String GIT_EXECUTABLE = "git";
```

Comment:
👍 

I'm slightly hesitant to flip this on for Windows w/o doing some testing first.  Happy to defer to you if you say bombs away though.